src/runnerSpec.ts

```diff
@@ -33,7 +33,11 @@ export function runnerSpec(suite: RunnerSuite, config: Config): () => void {
     const test = new RunnerSpec(folio, title, fn, suite);
     test._usedParameters = folio._pool.parametersForFunction(fn, `Test`, true);
     test.file = suite.file;
-    test.location = callLocation();
+    const location = callLocation();
```

Comment:
Let's also overwrite file here.

Reply:
Also, pass suite.file into location for the fall back so that you did not need to check return value.

Reply:
done